.github/workflows/deploy.yml

```diff
@@ -0,0 +1,19 @@
+name: Trigger ReproSchema to RedCap workflow
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Trigger workflow to build ReproSchema from RedCap
+        run: |
+          curl --request POST \
+          --header "Authorization: token ${{ secrets.B2AI_REDCAP2RS_PAT }}" \
```

Comment:
```suggestion
          --header "Authorization: token ${{ secrets.B2AIPREP_PAT }}" \
```

Reply:
The PAT is for the reposoitory redcap2rs(https://github.com/sensein/b2ai-redcap2rs) is is separate from the b2aiprep repository.